core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -254,11 +254,16 @@ export class RteHelperModalComponent {
         return;
       }
     } else if (this.componentId === this.COMPONENT_ID_TABS) {
-      // Value[0] corresponds to tab Contents.
-      for (let tab = 0; tab < value[0].length; tab++) {
-        if (value[0][tab].title === '' || value[0][tab].content === '') {
+      // Value[0] corresponds to all tabs Contents and titles.
```

Comment:
all tab contents and titles.

(tabs --> tab; do not capitalize "Contents")

Reply:
Done!